abis/FRAX.json

```diff
@@ -0,0 +1,595 @@
+[
```

Comment:
Would a standard ERC20 ABI work? Do we need a Frax specific ABI?

Reply:
yes, standard ERC20 should be good enough